### PR TITLE
fix: avoid watching too many files

### PR DESCRIPTION
**Pull request by davidfestal**
## Hey, I just made a Pull Request!

This fixes issue #20644

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: packages/backend-plugin-manager/src/manager/plugin-manager.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export class PluginManager implements BackendPluginProvider {
     config: Config,
     logger: LoggerService,
     preferAlpha: boolean = false,
-    mooduleLoader: ModuleLoader = new CommonJSModuleLoader(logger),
```
Comment (Rugvip) [inline packages/backend-plugin-manager/src/manager/plugin-manager.ts:52]: image

**Review by Rugvip — APPROVED** [review]
👍

**Comment by davidfestal** [comment]
@Rugvip What is expected to be done to have this PR merged ?

**Comment by Rugvip** [comment]
Just waiting for the release to go out!

File: packages/backend-plugin-manager/src/manager/plugin-manager.ts (hunk 1 above)
Comment (taras) [reply packages/backend-plugin-manager/src/manager/plugin-manager.ts:52]: ^^ moofactory

giphy-21

^^ moodule

**Pull request merged**